tools/k4aviewer/k4arecordingdockcontrol.cpp

```diff
@@ -392,7 +385,7 @@ bool K4ARecordingDockControl::PlaybackThreadFn(PlaybackThreadState *state)
         {
             if (image)
             {
-                image.set_timestamp(image.get_device_timestamp() + state->TimestampOffset);
+                image.set_timestamp(image.get_device_timestamp());
             }
```

Comment:
Does the viewer now even care to get and track TimestampOffset?

Reply:
The start offset is still displayed as part of the recording info, but it isn't needed for playback at all.

Reply:
Right, so it seems like dead code in the viewer

Reply:
I've removed all references to `state->TimestampOffset` already. `m_startTimestampOffsetUsec` was left in so that the value can still be displayed.